Parser/tokenizer.c

```diff
@@ -1100,8 +1101,7 @@ syntaxerror(struct tok_state *tok, const char *format, ...)
         goto error;
     }
 
-    args = Py_BuildValue("(O(OiiN))", errmsg,
-                         tok->filename, tok->lineno, offset, errtext);
+    args = Py_BuildValue("(O(OiiNii))", errmsg, tok->filename, tok->lineno, col_offset, errtext, tok->lineno, end_col_offset);
```

Comment:
Is not the line too long?

Reply:
`col_offset` and `end_col_offset` have type `Py_ssize_t`, but the format string contains `"i"` for them.